### PR TITLE
Add a note in upgrade-notes.md about the goog.inherits removal

### PR DESCRIPTION
**Pull request by fredj**
See #5194 

And please don't hesitate to correct my English ...

### CODE REVIEW

File: changelog/upgrade-notes.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 ## Upgrade notes
 
+#### Removal of goog.inherits
+
+goog.inherits is not used anymore in the library. Users extending OpenLayers classes and compiling their code with the library must now
+use ol.inherits instead. This also means that goog.base is no more available; the base class must now be called. If you previously had:
```
Comment (probins) [inline changelog/upgrade-notes.md]: /no more/no longer/

**Comment by ahocevar** [comment]
Do we need this? Neither `goog.base` nor `goog.inherits` are part of the OpenLayers API. And if people compile their applications together with the library, they should still be able to use both if they also use the Closure library. Or am I missing something?

File: changelog/upgrade-notes.md (hunk 1 above)
Comment (fredj) [inline changelog/upgrade-notes.md]: Thanks, fixed

**Comment by fredj** [comment]
> [...] they should still be able to use both if they also use the Closure library

Yes, it needs to be checked

**Comment by tschaub** [comment]
I agree this should not be needed as the `goog.inherits()` removal was an internal implementation detail only.

**Comment by fredj** [comment]
ok, closing

**Pull request closed without merging**